lib/bolt/plugin.rb

```diff
@@ -161,14 +157,16 @@ def initialize(config, pal, analytics)
       @unknown = Set.new
       @resolution_stack = []
       @unresolved_plugin_configs = config.plugins.dup
+      # The puppetdb plugin config comes from the puppetdb section, not from
+      # the plugins section
+      @unresolved_plugin_configs['puppetdb'] = config.puppetdb if config.puppetdb
```

Comment:
I doubt many people will run into this but should we raise an error if `puppetdb` is set under the `plugins` key?